docs/source/tips-and-tricks.rst

```diff
@@ -65,10 +132,3 @@ Here's a brief example: grouping integer by whether they are even or not.
 
     # True [((0,), {}), ((2,), {}), ((4,), {}), ((6,), {}), ((8,), {})]
     # False [((1,), {}), ((3,), {}), ((5,), {}), ((7,), {}), ((9,), {})]
-
-
-Conditional Execution on Cluster vs. Submit
```

Comment:
Did this get moved somewhere else? I have actually had a use for this. I think it's probably rarely going to be necessary, but this seems as good a place as any to stash it.

Reply:
Yeah, I show it as being in the same file but moved down a bit: https://github.com/htcondor/htmap/pull/191/files#diff-715600acf23027618322ceb355f10715R74-R78